ethereum/eth/src/main/java/org/hyperledger/besu/ethereum/eth/manager/task/AbstractRetryingPeerTask.java

```diff
@@ -147,6 +147,12 @@ protected boolean isPeerFailure(final Throwable error) {
     return error instanceof PeerBreachedProtocolException
         || error instanceof PeerDisconnectedException
         || error instanceof NoAvailablePeersException;
+    // should we add this here?:
+    //        || error instanceof IncompleteResultsException;
+  }
```

Comment:
? what are the implications of this

Reply:
This is something I need to talk to Karim about!

Reply:
Removed the comment.

Reply:
I think we should do as here 
https://github.com/hyperledger/besu/blob/main/ethereum/eth/src/main/java/org/hyperledger/besu/ethereum/eth/sync/tasks/RetryingGetHeaderFromPeerByHashTask.java#L106 it depends if we want to use AbstractRetryingSwitchingPeerTask or not because retrying with the same peers will not make sens